modules/swagger-codegen/src/main/resources/Java/pojo.mustache

```diff
@@ -140,15 +140,15 @@ public class {{classname}} {{#parent}}extends {{{parent}}} {{/parent}}{{#parcela
       return false;
     }{{#hasVars}}
     {{classname}} {{classVarName}} = ({{classname}}) o;
-    return {{#vars}}Objects.equals(this.{{name}}, {{classVarName}}.{{name}}){{#hasMore}} &&
+    return {{#vars}}{{#isByteArray}}Arrays{{/isByteArray}}{{#isBinary}}Arrays{{/isBinary}}{{^isByteArray}}{{^isBinary}}Objects{{/isBinary}}{{/isByteArray}}.equals(this.{{name}}, {{classVarName}}.{{name}}){{#hasMore}} &&
```

Comment:
@bmordue One suggestion: What about using `{{#isBinary}}java.util.Arrays{{/isBinary}}` to avoid unused import when no property is defined as byte array or binary?

Reply:
Maybe I've misunderstood, but I thought `isBinary` and `isByteArray` apply to a single CodegenProperty and would only be relevant inside `models.model.vars` in the template?

Reply:
My point is in https://github.com/swagger-api/swagger-codegen/pull/7341/files#diff-cc3e8eb9a2b26248932102a87608fc60R7, you're hard-coding "import java.util.Arrays;".

My suggestion is to avoid the hard-coding the import

Reply:
@cbornet what's your take on this?

Reply:
Yes there's a risk of having an unused import. But note that it's the case in a lot of other places of our templates.
Managing imports is really something not easy with code generators.

Reply:
Yup, we'll try to take care of the unused import in the future so this PR is good to merge.